Core/include/Acts/Geometry/VolumeBounds.hpp

```diff
@@ -66,6 +66,9 @@ class VolumeBounds {
     eOther = 6
   };
 
+  /// Static member to get the name of the BoundsType
+  static const std::vector<std::string> s_boundsTypeNames;
```

Comment:
`VolumeBounds` is already a base class with virtual methods. How about we add this ass a method derived classes just implement?

Reply:
I need access to the collection of names here, so I bundled them in the base class - this is for readability of some output file only.